app/src/features/Columns/ColumnSelect.tsx

```diff
@@ -41,11 +54,29 @@ const ColumnSelects = ({
   const { data: credentialOwners } = useApiOwnersListQuery({
     credential: credential?.[0].id,
   });
-  const { table, column } = pendingColumn;
-  const selectedOwner = credentialOwners?.[0];
+  const { table, column, owner: ownerId } = pendingColumn;
+  const selectedOwner = credentialOwners?.find(({ id }) => id === ownerId);
   const schema = selectedOwner?.schema as Record<string, string[]>;
+  const ownerTable =
+    table && selectedOwner
+      ? {
+          id: `${selectedOwner.id}/${table}`,
+          label: `${selectedOwner.name}/${table}`,
+        }
+      : undefined;
 
-  const tables = (schema && Object.keys(schema)) || [];
+  const tables = !credentialOwners
```

Comment:
Could we call this with something else like `tableOptions` ? To differentiate what we get from the column object and what we pass to the AutoComplete